directory-listing/common.css

```
@@ -17,14 +17,37 @@ body {
   border-bottom: 0.5rem solid #607d8b;
 }
 
+h1{
```

Use consistent spacing in rules same as others in all the files
```suggestion
h1 {
```